tests/w2w_test.py

```diff
@@ -1,50 +1,113 @@
 import os
 import pytest
-import rioxarray
+import rioxarray as rxr
 from affine import Affine
 import shutil
 from w2w.w2w import main
-from w2w.w2w import check_lcz_wrf_extent
+from w2w.w2w import _check_lcz_wrf_extent
+from w2w.w2w import _replace_lcz_number
+from w2w.w2w import check_lcz_integrity
 from w2w.w2w import wrf_remove_urban
 from w2w.w2w import create_wrf_gridinfo
 from w2w.w2w import calc_distance_coord
-import xarray
+import xarray as xr
+import numpy as np
 
 
 def test_argparse_shows_help():
     with pytest.raises(SystemExit):
         main(['--help'])
 
+def test_replace_lcz_number_ok(capsys):
+    info = {
+        'src_file': 'testing/Shanghai.tif',
+    }
+    LCZ_BAND = 0
+    lcz = rxr.open_rasterio(info['src_file'])[LCZ_BAND, :, :]
+
+    lcz_100 = np.array(
+        [1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
+         101, 102, 103, 104, 105, 106, 107]
+    )
+    lcz_new = _replace_lcz_number(
+            lcz=lcz,
+            lcz_to_change=lcz_100,
+        )
+
+    # Test whether LCZs 100+ are converted to 11+
+    assert (np.unique(lcz_new.data).tolist() ==
+            [1, 2, 3, 4, 5, 6, 8, 10, 11, 12, 14, 15, 17]) \
+           is True
+    # Make sure no 100+ LCZ classes are present
+    assert 101 not in np.unique(lcz_new.data)
+    assert 102 not in np.unique(lcz_new.data)
+    assert 104 not in np.unique(lcz_new.data)
+    assert 105 not in np.unique(lcz_new.data)
+    assert 107 not in np.unique(lcz_new.data)
```

Comment:
I think passing the check above, but failing on any of this is impossible, so we can remove those.

Reply:
True, its one or the other ... so I removed the last ones ...